api/home/for/data/ForecastDS/ForecastOfResponse/Configs/ExampleConfig.json

```diff
@@ -11,9 +11,34 @@
       "start": "2021-06-25 13:45:37.352574",
       "end": "2021-06-27 13:45:37.352574",
       "defaultVariables": [
-        "WaveHeight=0.12",
-        "WaveDirection=30",
-        "Thrusters=2"
+                                {
+              "type": "/Blueprints/Variable",
+              "name": "WaveHeight",
+              "value": "0.12",
```

Comment:
Could be nice to test this with different types and "valueType"
etc. "false", 1.23, "a_string value"